freud/diffraction.pyx

```diff
@@ -23,11 +23,248 @@ import freud.locality
 cimport numpy as np
 from libcpp cimport bool as cbool
 
+cimport freud._diffraction
+cimport freud.locality
 cimport freud.util
-from freud.util cimport _Compute
+from freud.util cimport _Compute, vec3
 
 logger = logging.getLogger(__name__)
 
+cdef class StaticStructureFactorDebye(_Compute):
+    r"""Computes a 1D static structure factor using the
+    Debye scattering equation.
+
+    This computes the static `structure factor
+    <https://en.wikipedia.org/wiki/Structure_factor>`__ :math:`S(k)` at given
+    :math:`k`-values by averaging over all :math:`k`-vectors of the same
+    magnitude. Note that freud employs the physics convention in which
+    :math:`k` is used, as opposed to the crystallographic one where :math:`q`
+    is used. The relation is :math:`k=2 \pi q`. The static structure factor
+    calculation is implemented using the Debye scattering equation:
+
+    .. math::
+
+        S(k) = \frac{1}{N} \sum_{i=0}^{N} \sum_{j=0}^{N} \text{sinc}(k r_{ij})
+
+    where :math:`N` is the number of particles, :math:`\text{sinc}` function is
+    defined as :math:`\sin x / x` (no factor of :math:`\pi` as in some
+    conventions). For more information see `this Wikipedia article
+    <https://en.wikipedia.org/wiki/Structure_factor>`__. The equation 4 from
+    the link can be obtained by replacing :math:`\frac{\sin(k r)}{kr}` with
+    :math:`\text{sinc}(k r)`. For a full derivation see :cite:`Farrow2009`.
```

Comment:
I'm not sure if I would reference a specific equation on Wikipedia -- this article could easily change, invalidating our documentation. Moreover, we have already defined the `sinc` function. I will remove this sentence.